<?xml version="1.0" encoding="UTF-8"?>
<!-- Phiên bản: 2.0 (Tái cấu trúc) -->
<!-- Vị trí: /ui/widgets/downloadwidget.ui -->
<ui version="4.0">
 <class>DownloadWidget</class>
 <widget class="QWidget" name="DownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="toolbarFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="DownloadToolbarWidget" name="toolbarWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="controlBarFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="DownloadControlBarWidget" name="controlBarWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="mainViewStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="listViewPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="downloadsTableView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="logViewPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="logTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="backToListButton">
         <property name="text">
          <string>Quay lại Danh sách</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="DownloadOptionsWidget" name="optionsWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DownloadToolbarWidget</class>
   <extends>QWidget</extends>
   <header>widgets/downloadtoolbarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DownloadControlBarWidget</class>
   <extends>QWidget</extends>
   <header>widgets/downloadcontrolbarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DownloadOptionsWidget</class>
   <extends>QWidget</extends>
   <header>widgets/downloadoptionswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
